frontend/src/components/SearchByCriteria.js

```diff
@@ -134,12 +150,34 @@ function SearchByCriteria() {
                   <Field className="ml-5" type="checkbox" name="picture" />
                 </h5>
               </div>
-              <button className="btn btn-success mt-3" type="submit">
+              <button
+                className="btn btn-success"
+                data-toggle="modal"
+                data-target="#criteriaStatusModal"
+                onClick={() => searchByValues(values)}
+                type="submit"
+              >
                 Keresés...
               </button>
+              <Modal status={status} id="criteriaStatusModal" />
             </Form>
           )}
         </Formik>
+
+        {recipes ? (
+          <ul className="list-group">
+            {recipes.map((recipe) => (
+              <li
+                key={recipe.id}
+                className="list-group-item list-group-item-action"
+              >
+                <div key={recipe.id}>{recipe.name}</div>
```

Comment:
Itt már felesleges a key.

Reply:
így is panaszkodik a keyekert  +/- 1 idk

Reply:
törölve